index.js

```diff
@@ -134,6 +135,20 @@ module.exports = function ExpressOpenApi (_routePrefix, _doc, _opts) {
     middleware.document = generateDocument(middleware.document, req.app._router || req.app.router)
     res.json(middleware.document)
   })
+
+  // OpenAPI document as yaml
+  router.get(new RegExp(`${routePrefix}.[(yaml)(yml)]`), (req, res) => {
+    const jsonSpec = generateDocument(middleware.document, req.app._router || req.app.router)
+    const yamlSpec = convertYaml(jsonSpec)
+
+    // no need if req.originalUrl is undefined or not, will match prefix.yaml or prefix.yml
+    const docType = req.originalUrl.split('.')[1]
```

Comment:
This is not quite enough to make the approach work, but also I think a more simple `res.type('yaml')` would be enough.  This call maps to [this underlying lookup](https://github.com/jshttp/mime-db/blob/master/db.json#L7884-L7886), and both are aliases to the same content-type.

Reply:
That was weird, I tried it with `'yaml'` before and it didn't work.

Reply:
Hm, what didn't work about it?

Reply:
No, as in I tried it, it didn't work, but it works now. Thanks for the suggestion though!